libs/order/src/selectors/order-all.selector.ts

```diff
@@ -0,0 +1,19 @@
+import { DaffOrder } from '../models/public_api';
+import { DaffOrderEntitySelectors, getDaffOrderEntitySelectors } from './order-entities.selector';
+import { DaffOrderSelectors, getOrderSelectors } from './order.selector';
+import { DaffOrderFeatureSelector, getDaffOrderReducersStateSelector } from './order-feature.selector';
+
+export interface DaffOrderAllSelectors<T extends DaffOrder> extends
+  DaffOrderEntitySelectors<T>,
+  DaffOrderSelectors,
+  DaffOrderFeatureSelector<T> {}
+
+export const getDaffOrderSelectors = (() => {
+  let cache;
+  return <T extends DaffOrder>(): DaffOrderAllSelectors<T> =>
```

Comment:
I think you should give the generic a default: T extends DaffOrder = DaffOrder

Reply:
oh yeah definitely. These were written before we were doing that.

Reply:
added default generic values everywhere in the selectors and also to the effects